css/style.css

```diff
@@ -541,3 +543,75 @@ nav {
     width: 100%;
   }
 }
+
+.modal-container {
+  display: none;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.6);
+  inset: 0 0;
+  width: 100vw;
+  height: 100vh;
+  padding: 10% 0;
+  margin: 0;
+}
+
+.modal {
+  background-color: #fff;
+  width: 90vw;
+  height: auto;
+  padding: 5%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 16px;
+}
+
+.link-container {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  justify-self: flex-end;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-close {
+  font-size: 28px;
+  color: #67798e;
+}
+
+.modal-close:hover,
+.modal-close:focus {
+  cursor: pointer;
+}
+
+#live-link {
+  background-image: url(../images/see-live.svg);
+}
+
+#source-link {
+  background-image: url(../images/github.svg);
+}
```

Comment:
- [ ] Please increase the font-weight for model buttons and reduce the gap between buttons and make sure it matches with Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168033883-91c4326f-af7d-4624-8a1f-a875e8a390f8.png)

### Figma design 

![image](https://user-images.githubusercontent.com/83803414/168033937-80099d98-e395-4173-be54-2166ee242fcf.png)